scripts/clear_chronos_user_id_and_submissions.py

```diff
@@ -0,0 +1,21 @@
+"""
+Clears the `chronos_user_id` field on all OSFUser instances. Needed for switching from one Chronos server to another.
+"""
```

Comment:
Will this just be a one time thing, or something we might have to run multiple times - debating over whether this should be in scripts/remove_after_use,

Reply:
I would imagine this to be something that's used more than once, if we need to switch from Chronos sandbox to staging on multiple OSF staging server.